staging/src/github.com/kubeedge/beehive/pkg/core/core.go

```diff
@@ -19,24 +19,13 @@ func StartModules() {
 	modules := GetModules()
 
 	for name, module := range modules {
-		var m common.ModuleInfo
-		switch module.contextType {
-		case common.MsgCtxTypeChannel:
-			m = common.ModuleInfo{
-				ModuleName: name,
-				ModuleType: module.contextType,
-			}
-		case common.MsgCtxTypeUS:
-			m = common.ModuleInfo{
-				ModuleName: name,
-				ModuleType: module.contextType,
-				// the below field ModuleSocket is only required for using socket.
-				ModuleSocket: common.ModuleSocket{
-					IsRemote: module.remote,
-				},
-			}
-		default:
-			klog.Exitf("unsupported context type: %s", module.contextType)
```

Comment:
Is it still needed to check for `unsupported context` for future extension? Though the contextType is hard code now..